appveyor.yml

```diff
@@ -0,0 +1,38 @@
+version: 1.0.{build}
+os: Visual Studio 2015
+clone_folder: C:\devel-src\roboptim-core
+environment:
+  CMAKE_INSTALL_PREFIX: C:\devel
+  PROJECT_SOURCE_DIR: C:\devel-src\roboptim-core
+  PATH: C:\devel\bin;%PATH%
+  PKG_CONFIG_PATH: C:\devel\lib\pkgconfig
+before_build:
+- cmd: >-
+    md C:\devel
+
+    if not exist C:\devel-src (md C:\devel-src)
+
+    cd C:\devel-src
+
+    choco install doxygen.portable
+
+    appveyor DownloadFile http://ftp.gnome.org/pub/gnome/binaries/win32/glib/2.28/glib_2.28.8-1_win32.zip
+
+    appveyor DownloadFile http://ftp.gnome.org/pub/gnome/binaries/win32/dependencies/pkg-config_0.26-1_win32.zip
+
+    appveyor DownloadFile http://ftp.gnome.org/pub/gnome/binaries/win32/dependencies/gettext-runtime_0.18.1.1-2_win32.zip
+
+    7z x glib_2.28.8-1_win32.zip -oC:\devel -r
+
+    7z x pkg-config_0.26-1_win32.zip -oC:\devel -r
+
+    7z x gettext-runtime_0.18.1.1-2_win32.zip -oC:\devel -r
+
+    cd %PROJECT_SOURCE_DIR%
+
+    git submodule update --init
+
+    cmake -G "Visual Studio 14 2015 Win64" -DCMAKE_INSTALL_PREFIX=%CMAKE_INSTALL_PREFIX%
```

Comment:
I gave it a go, but it seems to be [failing](https://ci.appveyor.com/project/bchretien/roboptim-core/build/1.0.6) on the dot-sourcing step:
```sh
. : The term './.travis/functions.ps1' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try 
again.
At line:2 char:3
+ . ./.travis/functions.ps1
+   ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (./.travis/functions.ps1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

We'll need to investigate this.